src/main/java/hudson/plugins/accurev/AccurevSCM.java

```diff
@@ -351,6 +359,10 @@ public String getDirectoryOffset() {
         return directoryOffset;
     }
 
+    public boolean isDeleteWorkspaceBeforeBuildStarts() {
+        return (deleteWorkspaceBeforeBuildStarts != null && deleteWorkspaceBeforeBuildStarts.equals("on")) ? true : false;
```

Comment:
This is handled by Jenkins with "Delete workspace before build starts" on jobs and through Pipeline with deleteDir()